togetherly/lib/views/widgets/chore_item.dart

```diff
@@ -18,7 +19,8 @@ class ChoreItem extends StatelessWidget {
     }
 
     bool isDueToday(Chore chore) {
-      return chore.dueDate == getDateToday();
+      return chore.dueDate.toString().split(' ')[0] ==
+          getDateToday().toString();
```

Comment:
Would it not be more readable and robust to define a helper method that takes two DateTime objects and compares `a.year == b.year && a.month == b.month && a.day == b.day`, rather than compare strings in this way?

Reply:
Compared to the longer statement of the toString().split(' ')[0], yes it may be more readable, but overall I disagree. I find it tedious to compare individual parts of a single object rather than the entire thing. If we later need more separation between what parts of the date we are comparing, then your suggestion would definitely make more sense, but for the moment I think this makes more sense.

Reply:
I will address this in some changes that I push.